### ISSUE TITLE
Mac OSX + Makefile Problem with find -daytime

### ISSUE THREAD

**Issue by bryanhanson** [cbeleites/hyperSpec#120]
Looks like Apple does not implement `find -daytime` in any shell they offer.  There are several possible workarounds that do roughly the same thing.  Here's the offending line in `Makefile`:

```
DESCRIPTION: $(shell find hyperSpec -maxdepth 1 -ctime 1 -not -ctime 0 -name "DESCRIPTION") #only if not modified today
```

One option is to replace `-daytime` with `-ctime 1` which checks if the file has been modified in the last 24 hrs.

**Comment by ximeg** [comment]
Closed after merged PR #112

**Comment by bryanhanson** [comment]
We are not getting the right dates from the changes I made.

**Comment by ximeg** [comment]
Well, actually if we address the sematic versioning issue #123 , we won't need to place current date into DESCRIPTION anymore. (It is also annoying to see the `hyperSpec/DESCRIPTION` file changed every time I build the package - git starts to complain about this file when I want to change the branch)

**Comment by bryanhanson** [comment]
Yes, I would like to see #123 fixed sooner rather than later.

**Comment by bryanhanson** [comment]
Alright, I'll close this as it's really going to be addressed with #123.
